morpheus/_lib/include/morpheus/messages/control.hpp

```diff
@@ -374,6 +389,7 @@ class ControlMessage
 
     nlohmann::json m_tasks{};
     nlohmann::json m_config{};
```

Comment:
Dont we want to be removing the JSON objects from this class and replacing them with `JSONValues`?